src/Tickets/Admin/Attendees/Provider.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * The main service provider for the Tickets Admin Attendees page.
+ *
+ * @since   TBD
+ * @package TEC\Tickets\Admin
+ */
+
+namespace TEC\Tickets\Admin\Attendees;
+
+/**
+ * Service provider for the Tickets Admin Attendees
+ *
+ * @since   TBD
+ * @package TEC\Tickets\Admin
+ */
+class Provider extends \tad_DI52_ServiceProvider {
```

Comment:
For new features, shouldn't we be using a `Controller` (extending `TEC\Common\Contracts\Provider\Controller`)?

Reply:
Could be, [the prototype is a year old](https://github.com/the-events-calendar/event-tickets/pull/2961/commits/3fe56f394eefabdd698837a3ade66adf41c3d6b6), that's the reason why this is structured this way.